CHANGELOG.md

```diff
@@ -917,7 +917,7 @@
 - help: show more configurable classes with --help-all, and use consistent naming [#331](https://github.com/voila-dashboards/voila/pull/331) ([@maartenbreddels](https://github.com/maartenbreddels))
 - Fix example use for config attribute resources [#323](https://github.com/voila-dashboards/voila/pull/323) ([@mkcor](https://github.com/mkcor))
 - Fix Windows develop install [#322](https://github.com/voila-dashboards/voila/pull/322) ([@vidartf](https://github.com/vidartf))
-- test: set timeout of pytest_tornado to 20 seconds to avoid false CI failured [#321](https://github.com/voila-dashboards/voila/pull/321) ([@maartenbreddels](https://github.com/maartenbreddels))
+- test: set timeout of pytest_tornado to 20 seconds to avoid false CI failed [#321](https://github.com/voila-dashboards/voila/pull/321) ([@maartenbreddels](https://github.com/maartenbreddels))
```

Comment:
Most likely the intended word was "failures"?

```suggestion
- test: set timeout of pytest_tornado to 20 seconds to avoid false CI failures [#321](https://github.com/voila-dashboards/voila/pull/321) ([@maartenbreddels](https://github.com/maartenbreddels))
```